certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/repository/StatisticsRepository.java

```diff
@@ -0,0 +1,11 @@
+package se.inera.intyg.certificateservice.domain.certificate.repository;
+
+import java.util.List;
+import java.util.Map;
+import se.inera.intyg.certificateservice.domain.unit.model.UnitStatistics;
+
+public interface StatisticsRepository {
+
+  Map<String, UnitStatistics> getStatisticsForUnits(List<String> unitIds,
```

Comment:
UnitIds should be HsaId instead of String? Call it issuedByUnitIds or something similar?